api/views.py

```diff
@@ -68,10 +58,40 @@ def create(self, request, **kwargs):
                 answer.status = Answer.CORRECT
                 puzzle.answer = answer.text
                 answer.save()
+                for meta in puzzle.metas.all():
```

Comment:
can we wrap this in a `transaction.on_commit()` block? Been trying to avoid long-running db transactions